index.js

```diff
@@ -42,9 +42,14 @@ const execa = require('execa');
 
 const generateCacheKey = async () => {
     const resolved = fs.readFileSync("Cartfile.resolved");
-    const {stdout} = await execa('xcodebuild', ['-version']);
-    const hash = crypto.createHash('sha1').update(resolved+stdout).digest('hex');
-    return `devbotsxyz:carthage:${process.env.GITHUB_REPOSITORY}:${hash}`;
+    try {
+        const {stdout} = await execa('xcodebuild', ['-version']);
+        const hash = crypto.createHash('sha1').update(resolved+stdout).digest('hex');
+        return `devbotsxyz:carthage:${process.env.GITHUB_REPOSITORY}:${hash}`;
```

Comment:
do you want to update the owner of this path ot be devcycle or nah

Reply:
its fine as is I think.